larpandoracontent/LArObjects/LArMCParticle.h

```diff
@@ -40,15 +40,39 @@ enum MCProcess
     MC_PROC_HAD_ELASTIC,
     MC_PROC_DECAY,
     MC_PROC_COULOMB_SCAT,
-    MC_PROC_UNUSED,
     MC_PROC_MU_BREM,
     MC_PROC_MU_PAIR_PROD,
     MC_PROC_PHOTON_INELASTIC,
     MC_PROC_HAD_IONI,
     MC_PROC_PROTON_INELASTIC,
     MC_PROC_PI_PLUS_INELASTIC,
     MC_PROC_CHIPS_NUCLEAR_CAPTURE_AT_REST,
-    MC_PROC_PI_MINUS_INELASTIC
+    MC_PROC_PI_MINUS_INELASTIC,
+    MC_PROC_TRANSPORTATION,
+    MC_PROC_RAYLEIGH,
+    MC_PROC_HAD_BREM,
+    MC_PROC_HAD_PAIR_PROD,
+    MC_PROC_ION_IONI,
+    MC_PROC_NEUTRON_KILLER,
+    MC_PROC_ION_INELASTIC,
+    MC_PROC_HE3_INELASTIC,
+    MC_PROC_ALPHA_INELASTIC,
+    MC_PROC_ANTI_HE3_INELASTIC,
+    MC_PROC_ANTI_ALPHA_INELASTIC,
+    MC_PROC_HAD_FRITIOF_CAPTURE_AT_REST,
+    MC_PROC_ANTI_DEUTERON_INELASTIC,
+    MC_PROC_ANTI_NEUTRON_INELASTIC,
+    MC_PROC_ANTI_PROTON_INELASTIC,
+    MC_PROC_ANTI_TRITON_INELASTIC,
+    MC_PROC_DEUTERON_INELASTIC,
+    MC_PROC_ELECTRON_NUCLEAR,
+    MC_PROC_PHOTON_NUCLEAR,
+    MC_PROC_KAON_PLUS_INELASTIC,
+    MC_PROC_KAON_MINUS_INELASTIC,
+    MC_PROC_HAD_BERTINI_CAPTURE_AT_REST,
+    MC_PROC_LAMBDA_INELASTIC,
+    MC_PROC_MU_NUCLEAR,
+    MC_PROC_TRITON_INELASTIC
```

Comment:
Is it worth adding an extra, final option just to allow algs to handle special cases, etc. Not mandatory, but something we've used in the past, e.g. for initialising variables, e.g. `HIT_CUSTOM `, `MC_CUSTOM` etc. [here](https://github.com/PandoraPFA/PandoraSDK/blob/master/include/Pandora/PandoraEnumeratedTypes.h).

Reply:
There is an MC_PROC_UNKNOWN which acts as a default if we see a process we don't recognise.